src/common/components/organisms/FidgetPickerModal.tsx

```
@@ -0,0 +1,305 @@
+
+
+
+  const getCategoryOptions = () => {
+    return fidgetOptions.filter(option => 
+      !selectedCategory || option.category === selectedCategory
+    );
+  };
+
+  return (
+    <div>
+      <style dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)